doc/how-to/microcloud-with-terraform.md

```diff
@@ -0,0 +1,62 @@
+# Single Node Microcloud deployment (Nested VMs)
+
+## Prerequisites
+
+- Single Ubuntu node with LXD installed
+- A network of type bridged on the host, e.g. from lxc network list
+```
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+|      NAME      |   TYPE   | MANAGED |      IPV4       |           IPV6            | DESCRIPTION | USED BY |  STATE  |
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+| br0            | bridge   | NO      |                 |                           |             | 17      |         |
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+```
+
+## Clone the terraform
+
+Get the terraform from the [demos folder](../../demos/microcloud-deploy)
+
+## Get started - init lxd - optional if already initialised
```

Comment:
As this is mainly the same as the [interactive tutorial](https://canonical-microcloud.readthedocs-hosted.com/en/latest/microcloud/tutorial/get_started/), can you please link to it instead?

It also shows how to install LXD if it's not already there.

Reply:
Having just `Get started` in the title should be enough.